### PR TITLE
Library port

### PR DESCRIPTION
**Pull request by gabe-l-hart**
## What this PR does / why we need it

This PR ports the internal code project!

## Special notes for your reviewer

This PR disables linting so that the code can be ported as-is. A follow-on PR will fix all linting errors and re-enable

## What gif most accurately describes how I feel towards this PR?
Example of a gif

### CODE REVIEW

**Comment by HonakerM** [comment]
@gabe-l-hart I'm excited for this PR! Lmk if/when you'd like a review

**Comment by gabe-l-hart** [comment]
@HonakerM Thanks! I think I finally have the build working, it would be great to get your review.

File: oper8/test_helpers/data/test_ca.crt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+-----BEGIN CERTIFICATE-----
```
Comment (HonakerM) [inline oper8/test_helpers/data/test_ca.crt:1]: Do we want to regen this cert to not use ibm.com?

File: .github/workflows/build-library.yml [hunk 2]
```diff
@@ -32,15 +32,19 @@ jobs:
...
+          # - setup: '3.12'
+          #   tox: 'py312'
 
     steps:
       - uses: actions/checkout@v3
       - name: Set up Python ${{ matrix.python-version.setup }}
         uses: actions/setup-python@v4
         with:
           python-version: ${{ matrix.python-version.setup }}
+      - name: Set up Node
```
Comment (HonakerM) [inline .github/workflows/build-library.yml:44]: Why do we still need node? I thought all of the cdk8s references were removed?

**Review by HonakerM — APPROVED** [review]

**Pull request merged**